config/forms/2021_2022.json

```diff
@@ -2004,9 +2674,18 @@
                       "value": "Prefer not to say"
                     }
                   }
-                },
+                }
+              },
+              "depends_on": [{
+                "details_known_8": "Yes"
+              }]
+            },
+            "person_8_economic_status": {
+              "header": "",
+              "description": "",
+              "questions": {
                 "ecstat8": {
-                  "check_answer_label": "Person 8’s Work",
+                  "check_answer_label": "Person 8's Work",
```

Comment:
This probably needs reverting? I think we want `’` consistently in all output

Reply:
reverted